src/main/kotlin/org/rust/ide/navigationToolbar/RsNavBarModelExtension.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2010-2019 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
```

Comment:
Wrong license comment

Reply:
Oops :sweat_smile:. That's what I get for copy pasting too much :)